tests/e2e-cypress/integration/legacyParamImport.cy.js

```diff
@@ -119,9 +119,9 @@ describe('Test on legacy param import', () => {
                 expect(kmlLayer.visible).to.be.true
             })
         })
-        it('is able to import an external KML from a legacy adminid query param', () => {
+        it('is able to import an external KML from a legacy adminId query param', () => {
             cy.goToMapView('en', {
-                adminid: adminId,
+                adminId: adminId,
```

Comment:
if you want to be even more concise, when the key and value are named the same you can just pass the value (it will automatically do the key: value for you).
```
cy.goToMapView('en', {
    adminId,
})
```